### PR TITLE
refactor: update config file to use pydantic

### LINKED ISSUE #70: Use pydantic setting for configurations
update configuration file to use pydantic configuration classes.

### PR DESCRIPTION
**Pull request by karelumair**
Update configuration to use class based configs

closes #70

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,9 @@
-MONGO_URI=mongodb://username:password@localhost:27017/db?authSource=admin
-MONGO_TEST_URI=mongodb://username:password@localhost:27017/test_db?authSource=admin
+MONGODB_HOST=localhost
+MONGODB_PORT=27017
+MONGODB_DATABASE=db_name
+MONGODB_USERNAME=username
+MONGODB_PASSWORD=password
+ENV=prod/test
```
Comment (pratikGhodke1) [inline .env]:
```suggestion
ENV="test" # Environments: "test", "prod", "dev"
```

File: config.py [hunk 2]
```diff
@@ -1,31 +1,48 @@
...
+    """Global Configurations"""
+
+    MONGODB_SETTINGS: list = [DatabaseConfig().dict()]
+    CELERY: dict = CeleryConfig().dict()
+
+
+class TestConfig(BaseConfig):
+    """Test Configurations"""
+
+    TESTING: bool = True
```
Comment (pratikGhodke1) [inline config.py]: Check if we can use dict as Field and load .env fields inside that. For example,
```python
class DatabaseConfig(BaseSettings):
    MONGODB_SETTINGS: list = [{
        HOST: Field(env="MONGODB_HOST"),
        . . .
    }]
```

**Review by pratikGhodke1 — CHANGES_REQUESTED** [review]
Check the comments

File: config.py (hunk 2 above)
Comment (karelumair) [reply config.py]: Actually this is not working, Last time I used os.environ.get() instead of Field. That's why it was working.

Comment (pratikGhodke1) [reply config.py]: We are anyways using ENV variables already set by bashrc/docker, then use `os.environ.get()`. Do you think `.env` will be any better than using already set ENV variables?

Comment (karelumair) [reply config.py]: Ok. No, I think bashrc will be fine.

Comment (pratikGhodke1) [reply config.py]: Use environ.get() then!

**Comment by karelumair** [comment]
Done

**Review by pratikGhodke1 — APPROVED** [review]

**Pull request merged**